deps/wrapper.h

```diff
@@ -14,13 +14,22 @@ using namespace ray;
 using ray::core::CoreWorkerProcess;
 using ray::core::CoreWorkerOptions;
 using ray::core::WorkerType;
+using ray::core::RayFunction;
+
+using namespace ray;
+using ray::core::CoreWorkerProcess;
+using ray::core::CoreWorkerOptions;
+using ray::core::RayFunction;
+using ray::core::TaskOptions;
+using ray::core::WorkerType;
```

Comment:
```suggestion
using ray::core::CoreWorkerProcess;
using ray::core::CoreWorkerOptions;
using ray::core::RayFunction;
using ray::core::TaskOptions;
using ray::core::WorkerType;
```

Reply:
Thanks. Looks like one of my rebases got messed up